### ISSUE TITLE
Is it possible to deploy and configure multipath in the plugin not on host?

### ISSUE THREAD

**Issue by zhucan** [IBM/ibm-block-csi-driver#111]

**Comment by 27149chen** [comment]
@zhucan , thank you for your suggestion, it is in the plan. I will update you when it is done.

**Comment by zhucan** [comment]
@27149chen Thanks

**Comment by zhucan** [comment]
@27149chen  Hello, I have a question, Do users need to configure multipath.conf by themselves?

**Comment by 27149chen** [comment]
@zhucan , when you are talking about "deploy and configure multipath", does it mean installing the multipath bundle and setting up the multipath.conf file?

**Comment by zhucan** [comment]
@27149chen yes, Maybe the default sets of the mutipath.conf is not suitable.

**Comment by 27149chen** [comment]
Sorry, I misunderstood your question before, about installing the multipath bundle and setting up the multipath.conf file, we are still under discussion. There are three options:
1. Let the k8s/platform admin to do that, which is what we did before.
2. Give a way outside the driver (maybe a script or in a container...) to do that. Because I think it is a host wide thing, the driver should not be given such permission.
3. In the plugin, what do you think of this way?

**Comment by zhucan** [comment]
@27149chen Thanks, I thinks set in the plugin is suitable, we can make the image with the configs.

**Comment by 27149chen** [comment]
But we if there are multiple csi drivers? our configuration may have impact on others.

**Comment by zhucan** [comment]
Is it compatible with others if we only use one multipath? when deploying csi drivers，if there is other storage has set multipath, can we only append it? @27149chen
